server/src/routes/api/v1/saladReviewsRouter.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express'
+import objection from 'objection'
+const{ ValidationError } = objection
+import { Review } from '../../../models/index.js'
+
+const saladReviewsRouter = new express.Router({ mergeParams: true })
+
+saladReviewsRouter.post('/', async (req, res) => {
+    const { body } = req
+    const saladIdParams = req.params.id
```

Comment:
I imagine the body of the review is required - we should pass the incoming data through `cleanUserInput` to ensure a body is provided and postgres does not save an empty string